backend/cms/models/site.py

```diff
@@ -22,14 +22,14 @@ class Site(models.Model):
         (ARCHIVED, 'Archived'),
     )
 
-    title = models.CharField(max_length=200)
-    name = models.URLField(max_length=60, unique=True)
+    name = models.CharField(max_length=200)
+    slug = models.CharField(max_length=200, unique=True)
     status = models.CharField(max_length=4, choices=STATUS)
-    language_tree = models.ForeignKey(LanguageTree, null=True, on_delete=models.SET_NULL)
+    languages = models.ManyToManyField(Language)
 
     events_enabled = models.BooleanField(default=True)
     push_notifications_enabled = models.BooleanField(default=True)
-    push_notification_channels = ArrayField(models.CharField(max_length=60))
+    push_notification_channels = ArrayField(models.CharField(max_length=60), blank=True)
```

Comment:
why should this be blank? Imo it should always have some value, maybe default to empty array? :thinking:

Reply:
I didn't want to add any complex UI elements to the region form at this stage, therefore I just went with a string-representation of the array and the user can just input different channels separated by spaces. The `RegionForm` then parses the string with a `.split(" ")`, which results on an empty array if the string is empty. The `blank`-parameter is not database-related, but just handles the form-validation instead. At some point at the future we should probably add an UI element to add and remove array elements in the backend form, but I don't think this has a high priority.

Reply:
Alright, I was on a wrong track somehow; I thought, `blank=true` allows the field to be null, but probably it only allows it to be an empty array, right?

Reply:
Yes, I think only `null=True` allows the field to be actually `null` in the database.